src/Symfony/Component/Security/Core/Security.php

```diff
@@ -67,6 +68,12 @@ public function isGranted($attributes, $subject = null): bool
 
     public function getToken(): ?TokenInterface
     {
-        return $this->container->get('security.token_storage')->getToken();
+        $tokenStorage = $this->container->get('security.token_storage');
+
+        if ($tokenStorage instanceof UsageTrackingTokenStorage && !$this->container->get('request_stack')->getMainRequest()) {
+            $tokenStorage->disableUsageTracking();
```

Comment:
shouldn't it be re-enabled after fetching the token? 
ie. `Security::getToken` is called just before handling a request

This would need to add a method in `UsageTrackingTokenStorage`
- `isTrackingEnabled`
- or `getTokenWithoutTracking`